index.html

```diff
@@ -42,7 +43,7 @@ <h1>
              </ul>
         </div>
         <section id="bellow">
-            <img src="./Images/Disabled.png"/>
+            <img src="./Images/Disabled.png" alt="Continue bellow"/>
```

Comment:
Just as I have described in the previous comment, this is also a decorative image and I suggest it is given a null value😉. `alt=" "`👍.